metrics/tests/site-startup.test.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from '@playwright/test';
+import { E2ESession } from '../../e2e/e2e-helpers';
+import MainSidebar from '../../e2e/page-objects/main-sidebar';
+import Onboarding from '../../e2e/page-objects/onboarding';
+import SiteContent from '../../e2e/page-objects/site-content';
+import { median } from '../utils';
+
+test.describe( 'Startup Metrics', () => {
+	const results: Record< string, number[] > = {};
+	const session = new E2ESession();
+	const siteName = 'Performance-Test-Site';
+
+	test.beforeAll( async () => {
+		await session.launch();
+
+		// Complete onboarding before tests
+		const onboarding = new Onboarding( session.mainWindow );
+		await expect( onboarding.heading ).toBeVisible();
+		await onboarding.continueButton.click();
+	} );
+
+	// eslint-disable-next-line no-empty-pattern
+	test.afterAll( async ( {}, testInfo ) => {
```

Comment:
I tried using a parameter with an underscore `_`, the linter was happy, but I got this error when running the metrics: `First argument must use the object destructuring pattern: _`. So we need this eslint exception 👍